graphql/schema.graphql

```diff
@@ -430,7 +431,7 @@ input ProjectSettingsInput {
 }
 
 input ProjectInput {
-  id: String!
+  id: String! @requireProjectAdmin
```

Comment:
I assume this is what affects the SaveProject mutation? So this would separately need to be added to the RepoRefInput, attachProjectToNewRepo, attachProjectToRepo, detachProjectFromRepo, forceRepotrackerRun inputs I think